portal-web/docroot/html/taglib/ui/input_permissions/page.jsp

```diff
@@ -119,11 +119,15 @@ String modelName = (String)request.getAttribute("liferay-ui:input-permissions:mo
 					<option <%= inputPermissionsViewRole.equals(RoleConstants.OWNER) ? "selected=\"selected\"" : "" %> value="<%= RoleConstants.OWNER %>"><liferay-ui:message key="owner" /></option>
 				</select>
 
-				<div class="mt-3 <%= inputPermissionsShowOptions ? "hide" : "" %>" id="<%= uniqueNamespace %>inputPermissionsShowOptionsLink" style="margin-bottom: 60px;">
-					<a class="btn btn-secondary btn-sm mt-3" href="javascript:<%= uniqueNamespace %>inputPermissionsShowOptions();" role="button"><liferay-ui:message key="more-options" /></a> <liferay-ui:icon-help message="input-permissions-more-options-help" />
+				<div>
```

Comment:
As a good practice, we should not use <div> tags without id or classes. In this case, we can use a <p> tag instead. 
Check that the prevoius <p> is not closed.

Reply:
Cool!